docs/master/sidebar.js

```diff
@@ -1,42 +1,93 @@
 module.exports = [{
-        title: 'Getting Started',
-        children: [
-            'getting-started/installation',
-            'getting-started/configuration',
-            'getting-started/tutorial',
-            'getting-started/migrating-to-lighthouse'
-        ]
-    },
+    title: 'Getting Started',
+    children: [
+        'getting-started/installation',
+        'getting-started/configuration',
+        'getting-started/tutorial',
+        'getting-started/migrating-to-lighthouse'
+    ]
+},
     {
         title: 'The Basics',
         children: [
             'the-basics/schema',
-            'the-basics/types',
             'the-basics/fields',
         ]
     },
     {
-        title: 'Guides',
+        title: 'Types',
         children: [
-            'guides/schema-organisation',
-            'guides/relay',
-            'guides/auth',
-            'guides/validation',
-            'guides/relationships',
-            'guides/file-uploads',
-            'guides/custom-directives',
-            'guides/error-handling',
-            'guides/plugin-development'
+            ['types/getting-started', 'Getting Started'],
+            'types/objects',
+            'types/scalars',
+            'types/enums',
+            'types/inputs',
+            'types/interfaces',
+            'types/unions',
+        ]
+    },
+    {
+        title: 'Queries',
+        children: [
+            ['queries/getting-started', 'Getting Started'],
+            'queries/pagination',
+            'queries/resolvers',
+            'queries/relay',
+        ]
+    },
+    {
+        title: 'Mutations',
+        children: [
+            ['mutations/getting-started', 'Getting Started'],
+            'mutations/resolvers',
+            'mutations/relay',
+        ]
+    },
+    {
+        title: 'Security',
+        children: [
+            ['security/getting-started', 'Getting Started'],
+            'security/authentication',
+            'security/authorization',
+            'security/validation',
+        ]
+    },
+    {
+        title: 'Directives',
+        children: [
+            ['directives/getting-started', 'Getting Started'],
+            ['directives/custom-directives', 'Creating Your Own'],
         ]
     },
     {
         title: 'Extensions',
         children: [
+            ['extensions/getting-started', 'Getting Started'],
```

Comment:
I think that each of those probably deserves it's own top-level entry.

We might have information in Extensions on how one could write their own, coupled with the Event system.